episodes/chap1_introduction_FL.md

```diff
@@ -87,18 +87,24 @@ Internet Movie Database (IMDB).
 
 ## Federated data analysis
 
-There are different ways in which privacy risks can be mitigated. We will focus on the idea of
-federated analysis. In a federated setting, the data with the data owner, who keeps full control
-over it. In this case, it is not the data that travels, but the analysis itself. The system sends
-a query or instruction to the data and only the results will get back to the user.
+There are different ways in which privacy risks can be mitigated. For example, a well-known technique
+is to send the data to a trusted third party (TTP). In this case, the data is sent to an trusted neutral
+party. The data can then be analyzed at that location in a traditional way. There are issues with
+this technique however. When the data is copied to the TTP, the original owner loses control over it. Another
+issue with it is that this technique results in a single point of failure. If the security at the TTP is
+breached, all the data it handled could be exposed.
+
+Instead, we will focus on the idea of federated analysis. In a federated setting, the data with the data owner,
```

Comment:
I think 'Instead' and 'the idea of' don't add much to the sentence's message. I would say "In this course, we will focus on federated analysis".